models/infrastructure_edge.py

```diff
@@ -185,15 +284,74 @@ class Vlan(BaseModel):
 
 
 CONTINENT_COUNTRIES = {
-    "North America": ["United States of America", "Canada"],
-    "South America": ["Mexico", "Brazil"],
-    "Africa": ["Morocco", "Senegal"],
-    "Europe": ["France", "Spain", "Italy"],
-    "Asia": ["Japan", "China"],
+    "North America": ["United States of America", "Canada", "Mexico", "Nicaragua", "Haiti", "Puerto Rico"],
+    "South America": ["Brazil", "South Africa", "Argentina", "Chile", "Peru", "Colombia", "Bolivia", "Ecuador"],
+    "Africa": [
+        "Morocco",
+        "Kuwait",
+        "Kenya",
+        "Egypt",
+        "Senegal",
+        "Republic of Congo",
+        "Ethiopia",
+        "Tanzania",
+        "Nigeria",
+        "Ghana",
+    ],
+    "Europe": [
+        "France",
+        "Spain",
+        "Italy",
+        "Finland",
+        "United Kingdom",
+        "Greece",
+        "Iceland",
+        "Germany",
+        "Norway",
+        "Cyprus",
+        "Lithuania",
+        "Serbia",
+        "Slovakia",
+        "Croatia",
+        "Bulgaria",
+        "Latvia",
+        "Poland",
+        "Hungary",
+        "Czech Republic",
+        "Montenegro",
+        "Bosnia and Herzegovina",
+        "Belgium",
+        "Estonia",
+        "Sweden",
+        "Denmark",
+        "Russia",
+        "Portugal",
+        "Netherlands",
+        "Switzerland",
+    ],
+    "Asia": [
+        "Japan",
+        "China",
+        "Taiwan",
+        "India",
+        "Nepal",
+        "Jordan",
+        "Qatar",
+        "Pakistan",
+        "Malaysia",
+        "South Korea",
+        "Turkey",
+        "United Arab Emirates",
+        "Singapore",
+        "Hong Kong",
+        "Thailand",
+    ],
     "Oceania": ["Australia", "New Zealand"],
 }
 
+# Chat GPT came up with the most cliche contact names here
 SITES = [
+    Site(name="sin", country="Singapore", city="Singapore", contact="Siti Tan"),
```

Comment:
I would prefer to keep the list of sites short to keep this file as easy to read as possible
the `site_generator` doesn't need that many sites, it will automatically increase the ID number if we need to generate more than 10 sites.

```
    site_names_generator(nbr_site=5)
        result >> ["atl1", "ord1", "jfk1", "den1", "dfw1"]

    site_names_generator(nbr_site=12)
        result >> ["atl1", "ord1", "jfk1", "den1", "dfw1", "iad1", "bkk1", "sfo1", "iah1", "mco1", "atl2", "ord2"]
```